pkg/registry/views/image-config.html

```diff
@@ -1 +1,28 @@
-Config stuff here
+<pre>{{ configCommand(config) }}</pre>
+<div class="row">
+    <dl class="col-xs-12 col-sm-12 col-md-4">
+        <dt translatable="yes">Run as</dt>
+        <dd ng-if="config.User">{{config.User}}</dd>
+        <dd ng-if="!config.User">root</dd>
```

Comment:
This might not be true, openshift for example runs containers as a made up user with a random uid.